sources/ITAddressBookMgr.h

```diff
@@ -139,6 +139,7 @@
 // Terminal
 #define KEY_DISABLE_WINDOW_RESIZING           @"Disable Window Resizing"
 #define KEY_PREVENT_TAB                       @"Prevent Opening in a Tab"
+#define KEY_DEFAULT_BG_ALPHA_ONLY             @"Only The Default BG Color Uses Transparency"
```

Comment:
Yes, let's change the constant too. It's confusing when the names don't match up, and long identifiers are a fact of life in objective c.